Session/Calls/Call Management/SessionCallManager.swift

```diff
@@ -136,12 +136,6 @@ public final class SessionCallManager: NSObject, CallManagerProtocol {
         func handleCallEnded() {
             WebRTCSession.current = nil
             UserDefaults.sharedLokiProject?.set(false, forKey: "isCallOngoing")
-            if CurrentAppContext().isInBackground() {
-                // Stop all jobs except for message sending and when completed suspend the database
-                JobRunner.stopAndClearPendingJobs(exceptForVariant: .messageSend) {
-                    NotificationCenter.default.post(name: Database.suspendNotification, object: self)
-                }
-            }
```

Comment:
So with this removed if there isn't a current call is this logic going to be run elsewhere? (ie. would it be possible for `reportCurrentCallEnded(reason:)` to be called when `currentCall` is null?

Reply:
Theoretically `reportCurrentCallEnded(reason:)` shouldn't be called when the `currentCall` is null, I'll need to review the logic and see if I can do better on this just in case.